docs/references/nextjs/current-user.mdx

```diff
@@ -5,7 +5,7 @@ description: Access the User object inside of your Server Components, Route Hand
 
 # `currentUser()`
 
-The `currentUser` helper returns the [`User`][user-object] object of the currently active user. This is the same [`User`][user-object] object that is returned by the [`useUser`](/docs/references/react/use-user) hook. However, it can be used in Server Components, Route Handlers, and Server Actions. Under the hood, this helper calls `fetch()` so it is automatically deduped per request.
+The `currentUser` helper returns the [`BAPI User`][user-object] object of the currently active user. However, it can be used in Server Components, Route Handlers, and Server Actions. Under the hood, this helper calls `fetch()` so it is automatically deduped per request.
```

Comment:
Do we use the acronym `BAPI` in the docs? If not, it might be clearer to say "Backend API" or just "Backend"